salt/minion.py

```diff
@@ -144,10 +144,10 @@ def resolve_dns(opts, fallback=True):
     if (opts.get('file_client', 'remote') == 'local' and
             not opts.get('use_master_when_local', False)):
         check_dns = False
+    # Because I import salt.log below I need to re-import salt.utils here
```

Comment:
Perhaps this comment could be worded a little better. Maybe something like: `Since salt.log is imported below, salt.utils.network needs to be imported here as well`?